src/MongoDB/BulkWrite.c

```diff
@@ -76,6 +76,10 @@ PHP_METHOD(BulkWrite, __construct)
 	}
 
 	intern->bulk = phongo_bulkwrite_init(ordered);
+
+	if (options && php_array_exists(options, "bypassDocumentValidation")) {
+		mongoc_bulk_operation_set_bypass_document_validation(intern->bulk, php_array_fetch_bool(options, "bypassDocumentValidation"));
+	}
```

Comment:
Although the patch is correct, I don't think it is complete. In the HHVM-116 ticket description it says "This means we need to modify BulkWrite, executeInsert(), and executeUpdate()." — Oh wait, we got rid of these two right? /me updates the HHVM-116 ticket.